server/test/unit/com/thoughtworks/go/server/service/GoConfigServiceTest.java

```diff
@@ -1135,11 +1135,33 @@ public void shouldSayThatAUserIsAuthorizedToViewAndEditTemplatesWhenTheUserHasPe
 
         assertThat(goConfigService.isAuthorizedToViewAndEditTemplates(new Username(templateAdminName)), is(true));
         assertThat(goConfigService.isAuthorizedToViewAndEditTemplates(new Username(secondTemplateAdminName)), is(true));
-        assertThat(goConfigService.isAuthorizedToViewAndEditTemplates(new Username(new CaseInsensitiveString(theSuperAdmin))), is(false));
+        assertThat(goConfigService.isAuthorizedToViewAndEditTemplates(new Username(new CaseInsensitiveString(theSuperAdmin))), is(true));
```

Comment:
Do you know why it was `false` earlier? That seems wrong. What were the consequences of that?